externs/shaka/player.js

```diff
@@ -1459,6 +1459,28 @@ shaka.extern.AdvancedAbrConfiguration;
  */
 shaka.extern.CmcdConfiguration;
 
+
+/**
+ * @typedef {{
+ *   enabled: boolean,
+ *   applyMaximumSuggestedBitrate: boolean,
+ *   estimatedThroughputWeightRatio: number
+ * }}
+ *
+ * @description
+ *   Common Media Server Data (CMSD) configuration.
+ *
+ * @property {boolean} enabled
+ *   If <code>true</code>, enables reading CMSD data in media requests.
+ *   Defaults to <code>true</code>.
+ * @property {boolean} applyMaximumSuggestedBitrate
+ *   Defaults to <code>true</code>.
+ * @property {number} estimatedThroughputWeightRatio
+ *   Defaults to <code>0.5</code>.
```

Comment:
This parameter definitely needs a description. Based on my reading of the code, I'd say something like:
"How much the estimatedThroughput of the CMSD data should be weighted against the default estimate, between 0 and 1."
But maybe you'd have a better description.

Reply:
Done!